src/js/BookReader/options.js

```diff
@@ -92,7 +92,10 @@ export const DEFAULT_OPTIONS = {
    * https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes ) */
   bookLanguage: null,
   
-  /** Fields used to populate the info window */
+  /**
+   * @type {Array<{label: string, value: *, extraValueClass: string?}>}
+   * Fields used to populate the info window
+   **/
   metadata: [],
```

Comment:
Where does this array get populated?

Reply:
Populated by the developer who's instantiating BR; added an example from the README to the docstring here.